.circleci/config.yml

```diff
@@ -59,6 +59,118 @@ jobs:
+        command: sudo rm -rf /usr/local/go
+    - run:
+        name: Install go
+        command: curl https://storage.googleapis.com/golang/go${GOVERSION}.darwin-amd64.tar.gz | sudo tar -C /usr/local -xz
+    - run:
+        name: List go version
+        command: go version
+    - run:
+        name: List go environment
+        command: go env
+    - run:
+        name: Create GOPATH bin directory
+        command: mkdir -p $GOPATH/bin
+    - run:
+        name: Download dep
+        command: curl -sL "https://github.com/golang/dep/releases/download/${GODEP_TAG}/dep-$(go env GOHOSTOS)-$(go env GOHOSTARCH)" -o /tmp/dep
+        environment:
+            GODEP_TAG: v0.4.1
+    - run:
+        name: Configure dep
+        command: mv /tmp/dep $GOPATH/bin/dep && chmod +x $GOPATH/bin/dep
+    - run:
+        name: Download github-release
+        command: |
+                curl -sL "https://github.com/aktau/github-release/releases/download/${GH_RELEASE_TAG}/$(go env GOHOSTOS)-$(go env GOHOSTARCH)-github-release.tar.bz2" \
+                -o /tmp/github-release.tar.bz2
+        environment:
+            GH_RELEASE_TAG: v0.7.2
+    - run:
+        name: Extract github-release
+        command: mkdir /tmp/github-release && tar -C /tmp/github-release/ -xj -f /tmp/github-release.tar.bz2
+    - run:
+        name: Configure github-release
+        command: mv /tmp/github-release/bin/$(go env GOHOSTOS)/$(go env GOHOSTARCH)/github-release $GOPATH/bin/github-release && chmod +x $GOPATH/bin/github-release
+    - run:
+        name: Install gh-release
+        command: go get -u github.com/progrium/gh-release/...
+    - run:
+        name: Make prerelease
+        command: cd $GOPATH/src/github.com/minishift/minishift && make prerelease
+    - run:
+        name: Make systemtray
+        command: cd $GOPATH/src/github.com/minishift/minishift && make cross_systemtray
+    - run:
+        name: Validate Commits
+        command: cd $GOPATH/src/github.com/minishift/minishift && make validate_commits
+    - run:
+        name: Make archives
+        command: |
+                mkdir -p $SOURCE_DIR/release
+
+                mkdir -p $BUILD_DIR/minishift-systemtray-$RELEASE_VERSION-darwin-amd64
+                cp $SOURCE_DIR/LICENSE $SOURCE_DIR/README.adoc $BUILD_DIR/darwin-amd64/systemtray/minishift $BUILD_DIR/minishift-systemtray-$RELEASE_VERSION-darwin-amd64
+                tar -zcf $SOURCE_DIR/release/minishift-systemtray-$RELEASE_VERSION-darwin-amd64.tgz -C $BUILD_DIR minishift-systemtray-$RELEASE_VERSION-darwin-amd64/
+
+                mkdir -p $BUILD_DIR/minishift-systemtray-$RELEASE_VERSION-windows-amd64
+                cp $SOURCE_DIR/LICENSE $SOURCE_DIR/README.adoc $BUILD_DIR/windows-amd64/systemtray/minishift.exe $BUILD_DIR/minishift-systemtray-$RELEASE_VERSION-windows-amd64
+                zip -r $SOURCE_DIR/release/minishift-systemtray-$RELEASE_VERSION-windows-amd64.zip $BUILD_DIR/minishift-systemtray-$RELEASE_VERSION-windows-amd64
```

Comment:
@anjannath have we tested all those steps in a fork to make sure it all wire up and we have systray specific binaries?

Reply:
@praveenkumar Yes, i tried it on my fork!